_posts/2024-04-27-fcmps.md

```diff
@@ -6,7 +6,7 @@ tags: [crypto, community]
 author: Luke "Kayaba" Parker
 ---
 
-Full-Chain Membership Proofs, as a concept, is a replacement for rings within the Monero protocol. While rings have offered sender privacy to Monero since it launched, they're vulnerable to attacks such as the [EAE attack](https://www.youtube.com/watch?v=iABIcsDJKyM&list=PLsSYUeVwrHBnAUre2G_LYDsdo-tD0ov-y&index=9&pp=iAQB), [have difficulties upon chain reorganzations](https://www.youtube.com/watch?v=6CVcirD90pg&list=PLsSYUeVwrHBnAUre2G_LYDsdo-tD0ov-y&index=4&pp=iAQB), and in general enable statistical analysis (mitigated by distribution of the decoy selection algorithm). Full-Chain Membership Proofs prove the output spent is one of _any_ output on the chain, effectively removing all of these risks. This means every input goes from an immediate anonymity set of 16 to 100,000,000.
+Full-Chain Membership Proofs, as a concept, is a replacement for rings within the Monero protocol. While rings have offered sender privacy to Monero since it launched, they're vulnerable to attacks such as the [EAE attack](https://www.youtube.com/watch?v=iABIcsDJKyM&list=PLsSYUeVwrHBnAUre2G_LYDsdo-tD0ov-y&index=9&pp=iAQB), [have difficulties upon chain reorganizations](https://www.youtube.com/watch?v=6CVcirD90pg&list=PLsSYUeVwrHBnAUre2G_LYDsdo-tD0ov-y&index=4&pp=iAQB), and in general enable statistical analysis (mitigated by distribution of the decoy selection algorithm). Full-Chain Membership Proofs prove the output spent is one of _any_ output on the chain, effectively removing all of these risks. This means every input goes from an immediate anonymity set of 16 to 100,000,000.
 
 Two proposals have been made for Monero offering such privacy, both under the "Full-Chain Membership Proofs" moniker (further mentions acronymed to "FCMPs"). The first was announced at [MoneroKon in 2023](https://www.youtube.com/watch?v=vrCAiLPfXlg), and was intended to be deployed with/after [Seraphis]({{ site.baseurl }}/2021/12/22/what-is-seraphis.html). Seraphis distinguished between "membership", the output spent is one of some outputs, and "spend authorization", the output being spent is authorized by the private key holder. With that, much more efficient proofs for membership became possible, including the "Grootle" proofs it was originally proposed with (effectively a ring of 128). FCMPs further improved upon this, requiring Seraphis's new key/transaction format to do so.
```

Comment:
```suggestion
Two proposals have been made for Monero offering such privacy, both under the "FCMP" moniker. The first was announced at [MoneroKon in 2023](https://www.youtube.com/watch?v=vrCAiLPfXlg), and was intended to be deployed with/after [Seraphis]({{ site.baseurl }}/2021/12/22/what-is-seraphis.html). Seraphis distinguished between "membership" - the output spent is one of some outputs; and "spend authorization" - the output being spent is authorized by the private key holder. With that, much more efficient proofs for membership became possible, including the "Grootle" proofs it was originally proposed with (effectively a ring of 128). FCMPs further improves upon this, requiring Seraphis's new key/transaction format to do so.
```

Reply:
The latter proposal was not called FCMP yet FCMP+SA+L, and later FCMP++.